src/main/content/_assets/js/instance-common.js

```diff
@@ -243,6 +285,14 @@ function setInstanceSelections(instances){
     return $firstInstance.find(".bx--accordion__title").text().trim();
 }
 
+function parseDateFromTimestamp(creationTimestamp){
+    let creationDate = new Date();
+    creationDate.setFullYear(creationTimestamp.year);
+    creationDate.setMonth(creationTimestamp.monthOfYear - 1);
+    creationDate.setDate(creationTimestamp.dayOfMonth);
+    return creationDate.toLocaleDateString();
+}
```

Comment:
shouldn't need to set all the fields here like this, you should be able to just pass the date into the Date constructor and call `toLocaleDateString`.

Reply:
should be able to do something like `return new Date(creationTimestamp).toLocaleDateString();`